gallery/src/components/NxButton/NxButtonPage.tsx

```diff
@@ -68,15 +68,18 @@ export default function NxButtonPage() {
                           id="nx-button-icon-example"
                           liveExample={NxButtonIconExample}
                           codeExamples={nxButtonIconCode}>
-        An example of a button containing an icon in addition to text.
+        An example of buttons containing icons. When an icon's only content consists of an icon it should be wrapped
+        in an <code className="nx-code">NxTooltip</code> and its <code className="nx-code">aria-label</code> set.
```

Comment:
This isn't true.  If the button has a tooltip, you shouldn't need to set the aria-label on anything.

Reply:
> This isn't true. If the button has a tooltip, you shouldn't need to set the aria-label on anything.

https://github.com/sonatype/sonatype-react-shared-components/pull/247/commits/e562b598f7b399b6a6f123cac9a02451fbd78726